CHANGELOG.md

```diff
@@ -3,7 +3,37 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.0dev - [date]
+## v1.1.0 - 01.07.2025
+
+Second release of nf-core/drugresponseeval.
+
+### `Added/Changed`
+
+- Added support for AWS: changed the structure of load response and parameter check to conform more to Nextflow
+  best practices.
+- Separate singularity container because conversion of Docker container took up too much space.
+
+### `Added`
+
+- Preprint is out now! Linking it in the documentation.
+- Added authors and licenses to the python scripts.
+- Added `no_hyperparameter_tuning` option for quick runs without hyperparameter tuning.
+- Flag `final_model_on_full data`: if True, a final/production model is saved in the results directory. If hyperparameter_tuning is true, the final model is tuned, too. The model can later be loaded using the implemented load functions of the drevalpy models.
+
+### `Changed`
+
+- Moved all publishDir directives to modules.config.
+- Fixed drevalpy versions in conda and docker to 1.3.5: now supporting Python 3.13
+
+### `Changed/Removed`
+
+- Simplified visualization: multiple short processes were creating overhang → more efficient in one process.
+
+### `Fixed`
+
+- Fixed errors that arose from the latest drevalpy version.
```

Comment:
This changelog does not seem to conform to standards. Multiple section heading repeated. Please see other nf-core changelogs for proper sections to be reported. It is best to be as detailed in commits, PR names and Changelog as possible which will make the auto release notes generation easier for you